wicket-core/src/main/java/org/apache/wicket/Session.java

```diff
@@ -184,6 +195,36 @@ public static Session get()
 		}
 	}
 
+	/**
+	 * Check if a BCP 47 / III language code indicates an RTL language, i.e.
+	 * either: - a language code explicitly specifying one of the right-to-left
+	 * scripts, e.g. "az-Arab", or
+	 * <p>
+	 * - a language code specifying one of the languages normally written in a
+	 * right-to-left script, e.g. "fa" (Farsi), except ones explicitly
+	 * specifying Latin or Cyrillic script (which are the usual LTR
+	 * alternatives).
+	 * <p>
+	 * <a href="http://www.unicode.org/iso15924/iso15924-num.html">
+	 * The list of right-to-left scripts appears in the 100-199 range in</a>, of which Arabic and
+	 * Hebrew are by far the most widely used. We also recognize Thaana, N'Ko,
+	 * and Tifinagh, which also have significant modern usage. The rest (Syriac,
+	 * Samaritan, Mandaic, etc.) seem to have extremely limited or no modern
+	 * usage and are not recognized. The languages usually written in a
+	 * right-to-left script are taken as those with 
+	 * <a href="http://www.iana.org/assignments/language-subtag-registry">Suppress-Script</a>:
+	 * Hebr|Arab|Thaa|Nkoo|Tfng, as well as
+	 * Sindhi (sd) and Uyghur (ug). The presence of other subtags of the
+	 * language code, e.g. regions like EG (Egypt), is ignored.
+	 *
+	 * @param locale - locale to check
+	 * @return <code>true</code> in case passed locale is right-to-left
+	 */
+	public static boolean isRtlLanguage(final Locale locale) {
```

Comment:
Any reason this method to be `public` ?
IMO we should start with `private` until there is a good reason to be part of the API

Reply:
method made private

Reply:
@martin-g can this method be made public? We are using it for utility purposes :)

Reply:
OK!